src/custodia/client.py

```diff
@@ -8,6 +8,7 @@
 from jwcrypto.jwk import JWK
 
 import requests
+from requests_gssapi import HTTPSPNEGOAuth
```

Comment:
I don't like the fact that all users have to install requests_gssapi. Please make the import optional and add an optional dependency to setup.py (``gssapi`` extras require).